deploy/KovanDeploy.js

```diff
@@ -88,12 +88,12 @@ module.exports = async function (hre) {
         contract: "OracleAdapter",
     })
 
-    // takes in the chainlink oracle wrapped (to return 10^18) and a gas price oracle
-    // not wrapped (to return 10^9)
+    // takes in the Fast Gas/Gwei and ETH/USD Chainlink oracles (not wrapped in adapter)
+    // provides the USD/Gas price in WAD format
     const gasPriceOracle = await deploy("GasPriceOracle", {
         from: deployer,
         log: true,
-        args: [ethOracleAdapter.address, gasOracle.address],
+        args: [ethOracle, gasOracle.address],
```

Comment:
Double sanity check: Why are we no longer wrapping the eth price oracle here?

Reply:
No longer wrapped because we assume that both oracles are given to the gasOracle as raw chainlink feeds (seen by the IChainlinkOracle interfaces used in gasOracle.sol). The gasOracle then ensures that it converts these raw chainlink answers to WAD format to conform to the IOracle standard.

Prior to this change the ethOracleAdapter was providing the gasOracle with WAD formatted answers, however it was incorrectly being passed through the IChainlinkOracle. Hence why it was called out during audit for potentially not being scaled.